### PR TITLE
[T4A5][F12-A2] Lee Jun Han Bryan

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review

### CODE REVIEW

**Comment by bryanleejh** [comment]
Not sure if the way I used enums is correct here. Still quite confused. Went to google it up but it's quite vague as well. Was contemplating representing the action using a String or Enum or maybe a boolean.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents addition or deletion of a tag in addressbook
+ */
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/tag/Tagging.java:7]: Glad to see header comments added =).

**Review by brandonyeoxg — COMMENTED** [review]
@bryanleejh Some comments added. Please close the PR after reading comments.

Perhaps you can think of implementing functionality to add and remove `Tagging`?

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 1 above)
Comment (bryanleejh) [reply src/seedu/addressbook/data/tag/Tagging.java:7]: Hahaha sorry will make it a habit (:

**Pull request closed without merging**